asmcomp/mach.mli

```diff
@@ -47,7 +51,6 @@ type operation =
   | Iconst_int of nativeint
   | Iconst_float of int64
   | Iconst_symbol of string
-  | Iconst_blockheader of nativeint
```

Comment:
It was supposed to make it easier for doing this sort of instrumentation (it was the approach used in the predecessor to Spacetime), but then it turned out that was kind of wrong.  So there's nothing to worry about here I don't think.